lib/multicell_location/Kconfig

```diff
@@ -20,7 +20,8 @@ if MULTICELL_LOCATION
 
 config MULTICELL_LOCATION_SERVICE_NRF_CLOUD
 	bool "nRF Cloud location service"
-	depends on MODEM_JWT
+	depends on NRF_CLOUD_MQTT || NRF_CLOUD_MQTT
```

Comment:
hmm, maybe this is a typo?  MQTT || MQTT?

Reply:
Hah, yep, I just saw that. I'll push again when I've actually tested